Cargo.toml

```diff
@@ -47,4 +47,4 @@ tokio-stream = "0.1"
 tracing = { version = "0.1", default-features = false }
 tracing-core = { version = "0.1", default-features = false }
 tracing-subscriber = { version = "0.3", default-features = false }
-url = { version = "2.5", default-features = false }
+url = { version = "2.5.2", default-features = false } #https://github.com/servo/rust-url/issues/992
```

Comment:
should we do a changelog entry for those crates affected?

Reply:
yes we should. Have updated the otel-sdk changelog.